radio/src/dataconstants.h

```diff
@@ -259,6 +259,11 @@ enum UartModes {
   UART_MODE_TELEMETRY,
   UART_MODE_SBUS_TRAINER,
   UART_MODE_LUA,
+//OW
```

Comment:
Could you please remove these?

Reply:
when you give your ok I'll sure will clean up

Reply:
If I asked you to come up with a PR, that means that's where we are heading ;-)